### ISSUE TITLE
Failed VPN authentication

### ISSUE THREAD

**Issue by naspass** [MarkusMcNugen/docker-qBittorrentvpn#25]
I'm using a .txt with torguard username and password on separate lines (when I didn't have this, I was getting an error). The .txt is referenced in the .ovpn file after "auth-user-pass". The username and password are correct; however, when I start the container, I get the following error:

`Mon Jun  3 07:35:00 2019 [TG-OVPN-CA] Peer Connection Initiated with [AF_INET]xx.xx.xx                                           

Mon Jun  3 07:35:02 2019 AUTH: Received control message: AUTH_FAILED                                                                          

Mon Jun  3 07:35:02 2019 SIGTERM[soft,auth-failure] received, process exiting `

**Comment by eyeruleall** [comment]
Did you ever get this solved? I'm having the same issue.

**Comment by naspass** [comment]
No, I ended up using a Deluge docker instead: https://hub.docker.com/r/binhex/arch-delugevpn/

**Comment by skyzo35** [comment]
I had this issue, but i fixed it by using the cred generated from my VPN provider for this connection instead of the VPN provider cred
